pmm/pmm2-server-autobuild.groovy

```diff
@@ -52,8 +57,17 @@ pipeline {
                     popd
 
                     git rev-parse --short HEAD > shortCommit
-                    echo "UPLOAD/${DESTINATION}/${JOB_NAME}/pmm/\$(cat VERSION)/${GIT_BRANCH}/\$(cat shortCommit)/${BUILD_NUMBER}" > uploadPath
+                    echo "UPLOAD/pmm2-components/yum/${DESTINATION}/${JOB_NAME}/pmm/\$(cat VERSION)/${GIT_BRANCH}/\$(cat shortCommit)/${BUILD_NUMBER}" > uploadPath
+                    cat VERSION > versionTag
                 '''
+
+                script {
+                    def versionTag = readFile('versionTag').trim()
```

Comment:
maybe we can just read `VERSION`?

Reply:
should work but it doesn't. Neither mine, nor just reading from VERSION